src/style.css

```
@@ -1 +1,95 @@
 /* add styles here */
+* {
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.form--big,
+.form--small {
```

```suggestion
.form {
```
that's why you have common `form` class - to apply shared styles

---

fix same everywhere